Description: We install into a private location, so no namespace_packages handling is necessary
Author: Stefano Rivera <stefanor@debian.org>
Forwarded: not-necessary
Last-Update: 2011-11-21
--- a/setup.py
+++ b/setup.py
@@ -42,7 +42,6 @@
           'beetsplug.bpd',
           'beetsplug.web',
       ],
-      namespace_packages=['beetsplug'],
       entry_points={
           'console_scripts': [
               'beet = beets.ui:main',
